docs/support/account.md

```diff
@@ -1,115 +1,32 @@
 # Account Management FAQs
+<details><summary>
+1. I am a vendor and my TechPass account has got locked. How to unlock my TechPass account?</summary>
+<br>Vendor's TechPass account will be locked after five consecutive unsuccessful login attempts. Go to <a href="https://passwordreset.microsoftonline.com/">re set password</a> and follow the on-screen instructions.</details><br>
+<details><summary>
+2. What is your password reset policy for TechPass accounts?</summary>
+<br>For vendors, we follow the <a href="https://docs.microsoft.com/en-us/azure/active-directory/authentication/concept-sspr-policy#administrator-password-policy-differences">password policy of Azure Active Directory</a>. Hence vendors will receive a password expiry notification accordingly. You can <a href="https://passwordreset.microsoftonline.com/">reset password</a> by following the on-screen instructions listed in this page.<br><br>
+However, if you are a public officer, you will be notified to reset your GSIB password as per the WOG password policies. If there are any issues with this, please contact your organisation's Agency Facility Management (AFM).</details><br>
+<details><summary>
+3. How often should I verify my security info?</summary>
+<br>As security information is vital, you need to make sure it is always up-to-date. You will receive a reminder every 180 days to review your security info and update it as needed.<br><br>
+To manage your security info any time, go to <a href="https://myaccount.microsoft.com/">My Account</a>.</details><br>
+<details><summary>
+4. I have lost the mobile device which I used for MFA authentications. What should I do?</summary>
 
-1. I am a vendor and my TechPass account has got locked. How to unlock my TechPass account?
+**If you are a vendor**:
 
-  Vendor's TechPass account will be locked after five consecutive unsuccessful login attempts. To unlock, go to https://passwordreset.microsoftonline.com/ and follow the on-screen instructions.
+i. Raise a [service request](https://go.gov.sg/techpass-sr) to contact our technical support to remove the MFA configured for your TechPass account.
 
-2. What is your password reset policy for TechPass accounts?
+ii. Once this done you will be notified. Proceed to [re-set-techpass-mfa-for-new-device](re-set-techpass-mfa-for-new-device) using your new mobile device.
```

Comment:
if there's a filename change, change this too: re-set-techpass-mfa-for-new-device

Reply:
done .